index.html

```diff
@@ -4,15 +4,264 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet"/>
   <link rel="stylesheet" href="styles/main.css" />
 
   <title>JT's Capstone One (Module one)</title>
 </head>
 <body>
-  <h1>Jud&aacute; Teixeira's Capstone One</h1>
-  <p>
-    Hi there
-  </p>
+  <div class="grid-container">
+    <header>  <!-- START OF HEADER -->
+      <div class="name-logo">
+        <a id="name-logo" href="#">
+          <img src="icons/icon.png" alt="Logo" />
+        </a>
+      </div>
+      <!-- Mobile Menu -->
+      <nav class="menu-mobile-container hide">
+        <a class="menu-x" id="menu-x" href="#">
+          <div>X</div>
+        </a>
+        <ul class="menu-mobile-list">
+          <li class="main-btn"><a href="#Main">Main</a></li>
+          <li class="about-btn"><a href="#About">About</a></li>
+          <li class="archive-btn"><a href="#Archive">Archive</a></li>
+          <li class="footer-btn"><a href="#Footer">ECM ID</a></li>
+        </ul>
+      </nav>
+
+      <!-- Desktop Menu -->
+      <div class="menu-desktop">
+        <a class="menu-hamb" href="#">
+          <img id="menu-hamb" src="icons/menu.png" alt="Menu" />
+        </a>
+        <ul class="menu-desktop-item">
+          <li><a href="#Main">Main</a></li>
+          <li><a href="#About">About</a></li>
+          <li><a href="#Archive">Archive</a></li>
+          <li id="no-padding"><a class="cc-footer" id="cc-footer" href="#Footer">ECM ID</a></li>
+        </ul>
+      </div>
+    </header>  <!-- END OF HEADER -->
+
+    <div class="menu-mobile-bg hide">
+      
+    </div>
+
+    <section class="home" id="Home">  <!-- START OF HOME -->
+      <div class="home-heading">
+        <span>
+          "Hello! Electronic Music world"
+        </span>
+        <h1>
+          EDM MARKETING <br>
+          CONFERENCE <br>
+          2022
+        </h1>
+        <p>
+          A internactional conference that unites 
+          the forward thinkers on EDM marketing  
+          with people from over 80 countries 
+          is taking place in March, in Brazil.
+        </p>
+        <div class="event-details">
+          <h2>
+            2022.03.10(THU) ~ 11(FRI)
+          </h2>
+          <p class="event-location">
+            @ Arena das Dunas, Natal 
+            Gramado e parte do estacionamento
+          </p>
+        </div>
+        
+      </div>
+    </section> <!-- END OF HOME -->
+
+    <section class="main" id="Main"> <!-- START OF MAIN -->
+      <div class="h3-header">
```

Comment:
- [ ] Great use of HTML semantic tag, I salute your effort, however, please play with some adjustments to align your `p` texts images on the same level, see this image below, there is a kind of vertical displacement between them, don't you think so? alright, let us fix that 😄 .

-  <img width="256" alt="CARD" src="https://user-images.githubusercontent.com/79058364/157603259-e472b4d4-d458-4deb-a3f0-da25f38e5b27.png">